src/main/scala/net/snowflake/spark/snowflake/CloudCredentialsUtils.scala

```diff
@@ -55,17 +60,36 @@ private[snowflake] object AWSCredentialsUtils {
        |""".stripMargin.trim
   }
 
-  def getCreds(sqlContext: SQLContext,
-               params : MergedParameters) : AWSCredentials = {
+  def getSnowflakeCredentialsStringForAzure(
+        creds : StorageCredentialsSharedAccessSignature) : String = {
+    val sasToken = creds.getToken()
```

Comment:
Just to be safe, I assume Azure tokens use only alphanumeric chars, so no escaping needed?

Reply:
The SAS token will be something like below. AFAIK, the characters doesn't need any escaping. 
?sv=2016-05-31&ss=b&srt=sco&sp=rwdlac&se=2018-09-13T00:27:47Z&st=2017-06-19T16:27:47Z&spr=https,http&sig=aJc9WMofMGM2r90OO8Lw%2BzP%2BWC%2FwxDeP3ESf64JEfro%3D